### PR TITLE
xidel 0.9.8 (new formula)

### PR DESCRIPTION
**Pull request by benibela**
Add xidel

Not tested since I do not have a mac

- [+] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [+] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [-] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [?] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by paul-hammant** [comment]
I can confirm this works for me via:

```
brew install https://raw.githubusercontent.com/benibela/homebrew-core/d6dedb62d78832db389ec1fe0903f2b509ebe1cc/Formula/xidel.rb
```

File: Formula/xidel.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Xidel < Formula
+  desc "XPath/XQuery 3.0, JSONiq interpreter to extract data from HTML/XML/JSON"
+  homepage "http://www.videlibri.de/xidel.html"
+  url "https://github.com/benibela/xidel/releases/download/Xidel_0.9.8/xidel-0.9.8.src.tar.gz"
+  sha256 "72b5b1a2fc44a0a61831e268c45bc6a6c28e3533b5445151bfbdeaf1562af39c"
+
+  bottle :unneeded
```
Comment (woodruffw) [inline Formula/xidel.rb]: You can remove this line (this formula builds binaries, and as such would benefit from a bottle).

**Review by woodruffw — CHANGES_REQUESTED** [review]

File: Formula/xidel.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  depends_on "bash"
+  depends_on "fpc"
+  depends_on "openssl" => :recommended
+
+  def install
+    cd buildpath/"programs/internet/xidel" do
+      system "bash", "build.sh"
```
Comment (bfontaine) [inline Formula/xidel.rb]: `system "./build.sh"` should be enough.

File: Formula/xidel.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+class Xidel < Formula
+  desc "XPath/XQuery 3.0, JSONiq interpreter to extract data from HTML/XML/JSON"
+  homepage "http://www.videlibri.de/xidel.html"
+  url "https://github.com/benibela/xidel/releases/download/Xidel_0.9.8/xidel-0.9.8.src.tar.gz"
+  sha256 "72b5b1a2fc44a0a61831e268c45bc6a6c28e3533b5445151bfbdeaf1562af39c"
+
+  bottle :unneeded
+
+  depends_on "bash"
```
Comment (bfontaine) [inline Formula/xidel.rb]: Is this really necessary?

**Review by woodruffw — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @benibela for the pull request!

**Pull request closed without merging**